ReleaseHistory.md

```diff
@@ -15,6 +15,25 @@
 * PRF: Add support for efficiently peeking into non-seekable streams for binary/text categorization.
 * NEW: Add a new `--timeout-in-seconds` parameter to `AnalyzeOptionsBase`, which will override the `TimeoutInMilliseconds` property in `AnalyzeContextBase`.
 * NEW: `--post-uri` will skip sending the SARIF log to the configured endpoint if the file contains no results or fatal execution errors.
+* NEW: Add below rules: 
+  `ADO1011.ReferenceFinalSchema`,
+  `ADO1013.ProvideRequiredSarifLogProperties`,
+  `ADO1014.ProvideRequiredRunProperties`,
+  `ADO1015.ProvideRequiredResultProperties`,
+  `ADO1016.ProvideRequiredLocationProperties`,
+  `ADO1017.ProvideRequiredPhysicalLocationProperties`,
+  `ADO1018.ProvideRequiredToolProperties`,
+  `ADO2008.ProvideRequiredSarifLogProperties`,
+  `ADO2012.ProvideRequiredReportingDescriptorProperties`,
+  `GH1011.ReferenceFinalSchema`,
+  `GH1013.ProvideRequiredSarifLogProperties`,
+  `GH1014.ProvideRequiredRunProperties`,
+  `GH1015.ProvideRequiredResultPropertties`,
+  `GH1016.ProvideRequiredLocationProperties`,
```

Comment:
Remove extra "t" from "Propertties"

Reply:
ouch. great find, I will fix the code and the release note. @EasyRhinoMSFT

Reply:
fixed.